pkgs/servers/openvscode-server/default.nix

```diff
@@ -164,15 +164,19 @@ in stdenv.mkDerivation rec {
     ln -s ${nodejs}/bin/node $out
   '';
 
-  meta = with lib; {
+  passthru.tests = {
+    inherit (nixosTests) openvscode-server;
+  };
+
+  meta = {
```

Comment:
`with lib;` in the package's `meta` is fine.
The mentioned #229030 seems to care only about `with lib;` in nixos modules, not packages.

Please undo :)
And sorry for the confusion^^

Reply:
Fixed!